Build fails on OpenBSD due to unsupported libfuzzer

**Description**
I'm trying to build `wgpu` API on OpenBSD current version, amd64 architecture. But **the build with Rust 1.96.0 fails due to unsupported `libfuzzer`** on this platform. See https://llvm.googlesource.com/libfuzzer/+/refs/heads/main/FuzzerPlatform.h

**Repro steps**
- Clone wgpu sources from GitHub with Git
- Build `wgpu` on OpenBSD/amd64 with Rust 1.96.0 (installed via OpenBSD package via `pkg_add rust`)
```sh
$ cargo build -v
(....)
   Compiling ureq-proto v0.6.0
       Running `rustc --crate-name ureq_proto --edition=2024 /home/fox/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/ureq-proto-0.6.0/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts,future-incompat --diagnostic-width=273 --crate-type lib --e       mit=dep-info,metadata,link -C embed-bitcode=no -C debuginfo=2 --cfg 'feature="client"' --check-cfg 'cfg(docsrs,test)' --check-cfg 'cfg(feature, values("client", "default", "server"))' -C metadata=81520c4408827186 -C extra-filename=-55d6db2a5cf256c0 --out-dir /home/f       ox/tmp/wgpu.git/target/debug/deps -L dependency=/home/fox/tmp/wgpu.git/target/debug/deps --extern base64=/home/fox/tmp/wgpu.git/target/debug/deps/libbase64-ae5c8ef0bbf79964.rmeta --extern http=/home/fox/tmp/wgpu.git/target/debug/deps/libhttp-14b79a8497dccb0f.rmeta -       -extern httparse=/home/fox/tmp/wgpu.git/target/debug/deps/libhttparse-6f8b5d5ec10e8124.rmeta --extern log=/home/fox/tmp/wgpu.git/target/debug/deps/liblog-3bbc3f94578cf45b.rmeta --cap-lints allow`
       Running `/home/fox/tmp/wgpu.git/target/debug/build/rayon-core-ddf88901fc880f16/build-script-build`
       Running `/home/fox/tmp/wgpu.git/target/debug/build/portable-atomic-4f98177526e9c801/build-script-build`
       Running `/home/fox/tmp/wgpu.git/target/debug/build/nv-flip-sys-c48bc112c5d3a0bc/build-script-build`
  warning: libfuzzer-sys@0.4.7: In file included from libfuzzer/FuzzerCrossOver.cpp:12:
  warning: libfuzzer-sys@0.4.7: In file included from libfuzzer/FuzzerMutate.h:15:
  warning: libfuzzer-sys@0.4.7: In file included from libfuzzer/FuzzerDictionary.h:16:
  warning: libfuzzer-sys@0.4.7: In file included from libfuzzer/FuzzerUtil.h:14:
  warning: libfuzzer-sys@0.4.7: In file included from libfuzzer/FuzzerBuiltins.h:14:
  warning: libfuzzer-sys@0.4.7: libfuzzer/FuzzerPlatform.h:72:2: error: "Support for your platform has not been implemented"
  warning: libfuzzer-sys@0.4.7:    72 | #error "Support for your platform has not been implemented"
  warning: libfuzzer-sys@0.4.7:       |  ^
  warning: libfuzzer-sys@0.4.7: 1 error generated.
  warning: libfuzzer-sys@0.4.7: ToolExecError: command did not execute successfully (status code exit status: 1): LC_ALL="C" "c++" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-gdwarf-2" "-fno-omit-frame-pointer" "-m64" "--target=x86_64-unknown-openbsd" "-Wa  ll" "-Wextra" "-std=c++17" "-fno-omit-frame-pointer" "-w" "-o" "/home/fox/tmp/wgpu.git/target/debug/build/libfuzzer-sys-b643b422c5084281/out/e5f0d71fb86e9d6b-FuzzerCrossOver.o" "-c" "libfuzzer/FuzzerCrossOver.cpp"cargo:rerun-if-env-changed=CXX_x86_64-unknown-openbsd
  (...)
```

**Expected vs observed behavior**
Build on OpenBSD/amd64 will succeed if `libfuzzer` is disabled in `naga/fuzz/` sources (`Cargo.toml` and `build.rs` files).

**Platform**
- OpenBSD current/amd64
```sh
$ uname -mrs
OpenBSD 7.9 amd64
```
- Rust v1.96.0 on OpenBSD installed via `rustc` package
```sh
$ rustc -vV
rustc 1.96.0 (ac68faa20 2026-05-25) (built from a source tarball)
binary: rustc
commit-hash: ac68faa20c58cbccd01ee7208bf3b6e93a7d7f96
commit-date: 2026-05-25
host: x86_64-unknown-openbsd
release: 1.96.0
LLVM version: 21.1.8
```

- `wgpu` sources from GitHub, `trunk` branch.

--- a/naga/fuzz/Cargo.toml
+++ b/naga/fuzz/Cargo.toml
@@ -10,7 +10,7 @@ build = "build.rs"
 [package.metadata]
 cargo-fuzz = true
 
-[target.'cfg(not(any(target_arch = "wasm32", target_os = "ios")))'.dependencies]
+[target.'cfg(not(any(target_arch = "wasm32", target_os = "ios", target_os = "openbsd")))'.dependencies]
 arbitrary = { workspace = true, features = ["derive"] }
 libfuzzer-sys.workspace = true
 

--- a/naga/fuzz/build.rs
+++ b/naga/fuzz/build.rs
@@ -1,6 +1,6 @@
 fn main() {
     cfg_aliases::cfg_aliases! {
-        fuzzable_platform: { not(any(target_arch = "wasm32", target_os = "ios", all(windows, target_arch = "aarch64"))) },
+        fuzzable_platform: { not(any(target_arch = "wasm32", target_os = "ios", target_os = "openbsd", all(windows, target_arch = "aarch64"))) },
     }
     // This cfg provided by cargo-fuzz
     println!("cargo::rustc-check-cfg=cfg(fuzzing)");

